components/table/table-controls.js

```diff
@@ -21,7 +21,8 @@ class TableControls extends SelectionControls {
 		return [super.styles, css`
 			:host {
 				--d2l-selection-controls-background-color: var(--d2l-table-controls-background-color);
-				z-index: 2; /* must be greater than d2l-table rows */
+				--d2l-selection-controls-shadow-display: var(--d2l-table-controls-shadow-display);
+				z-index: 5; /* Must be greater than d2l-table-wrapper and d2l-scroll-wrapper */
```

Comment:
There's a small `z-index` collision here where the scroll-wrapper's controls (left/right scroll buttons) can get hidden behind the table controls. But on the flip side, the controls need to be on top or else things like the overflow dropdown get hidden.

I think the only solution would be to have those scroll-wrapper buttons stick below the controls, instead of the top of the screen...

Reply:
I found a real simple fix for this actually!

![image](https://user-images.githubusercontent.com/8449639/219720896-9d81a426-c31c-4c35-9cd3-6fbd555cbb29.png)

Going to add it in a subsequent PR though, don't want to complicate this PR any further.